snuba/subscriptions/scheduler.py

```diff
@@ -50,14 +51,19 @@ def __init__(
         store: SubscriptionDataStore,
         partition_id: PartitionId,
         cache_ttl: timedelta,
+        metrics: MetricsBackend,
     ) -> None:
         self.__store = store
         self.__cache_ttl = cache_ttl
         self.__partition_id = partition_id
+        self.__metrics = metrics
+
         self.__subscriptions: List[Subscription] = []
         self.__last_refresh: Optional[datetime] = None
 
-    def __get_subscriptions(self, current_time: datetime) -> List[Subscription]:
+    def __get_subscriptions(self) -> List[Subscription]:
+        current_time = datetime.now()
```

Comment:
Passing the current time relative to the tick instead of wall clock time means this schedule would have been refreshed unnecessarily frequently during a backlog scenario. We only store the latest state of the schedule (there is no time dimension) there's no real benefit to anchoring this to the partition time rather than wall clock time.